wallet/src/de/schildbach/wallet/ui/MainActivity.kt

```diff
@@ -220,13 +220,6 @@ class MainActivity : AbstractBindServiceActivity(), ActivityCompat.OnRequestPerm
 
     private fun showContacts(mode: Int = MODE_SEARCH_CONTACTS) {
         bottom_navigation.menu.findItem(R.id.contacts)?.isChecked = true
-
-        val contactsFragment = ContactsFragment.newInstance(mode)
-        if (mode == MODE_VIEW_REQUESTS) {
-            addFragment(contactsFragment)
-        } else {
-            replaceFragment(contactsFragment)
-        }
```

Comment:
Clicking "view all requests" on the Contacts screen results in a "Contact Request" screen with no entries even when there contact requests are visible on the Contacts Screen.  Not sure if this is related.

Reply:
This issue is present in the current `evonet-develop` branch